backend/alembic/versions/0022_collections_.py

```diff
@@ -109,6 +78,39 @@ def upgrade() -> None:
                 "id": rom.id,
             },
         )
+
+    op.create_table(
```

Comment:
is it worth checking that the table doesn't exist before trying to create it? is that a bad idea?

Reply:
True! forgot about it. I just added the check